ncrypt/crypt_gpgme.c

```diff
@@ -1424,12 +1424,14 @@ static void show_one_recipient(struct State *state, gpgme_recipient_t r)
 
 /**
  * show_encryption_info - Show encryption information
+ * @param state  State to write to
+ * @param result Encryption recipients
```

Comment:
Actually, `result` contains slightly more than the recipients.  We're not using the extra info, but there is (and we may want to expand this block in the future to show it).

```c
$ grepc gpgme_decrypt_result_t /usr/include/
/usr/include/gpgme.h:typedef struct _gpgme_op_decrypt_result *gpgme_decrypt_result_t;
```
```c
$ grepc _gpgme_op_decrypt_result /usr/include/
/usr/include/gpgme.h:struct _gpgme_op_decrypt_result
{
  char *unsupported_algorithm;

  /* Key should not have been used for encryption.  */
  unsigned int wrong_key_usage : 1;

  /* True if the message was encrypted in compliance to the de-vs
   * mode.  */
  unsigned int is_de_vs : 1;

  /* The message claims that the content is a MIME object.  */
  unsigned int is_mime : 1;

  /* The message was made by a legacy algorithm without any integrity
   * protection.  This might be an old but legitimate message. */
  unsigned int legacy_cipher_nomdc : 1;

  /* Internal to GPGME, do not use.  */
  int _unused : 28;

  gpgme_recipient_t recipients;

  /* The original file name of the plaintext message, if
   * available.  */
  char *file_name;

  /* A textual representation of the session key used to decrypt the
   * message, if available */
  char *session_key;

  /* A string with the symmetric encryption algorithm and mode using
   * the format "<algo>.<mode>".  */
  char *symkey_algo;
};
```

I would say "Decryption result" in the comment.

Reply:
@flatcap